rollup.config.js

```diff
@@ -60,14 +60,17 @@ const buildUmd = ({ env }) => ({
     sourceMaps(),
     sizeSnapshot(),
     env === 'production' &&
-      uglify({
+      terser({
+        sourcemap: true,
         output: { comments: false },
         compress: {
           keep_infinity: true,
           pure_getters: true,
         },
         warnings: true,
+        ecma: 5,
         toplevel: false,
+        mangle: false,
```

Comment:
Why not mangle? This is expected from production umd I think.

Reply:
What options should we use for mangle?

Reply:
No options. Just remove `mangle: false`. Or is there a reason for it?

Reply:
No idea. Just wanted to check.